stripe.go

```diff
@@ -450,9 +457,20 @@ func (s *BackendImplementation) Do(req *http.Request, body *bytes.Buffer, v inte
 
 // ResponseToError converts a stripe response to an Error.
 func (s *BackendImplementation) ResponseToError(res *http.Response, resBody []byte) error {
+
```

Comment:
Ultra minor, but would you mind taking out this extraneously added newline?

Reply:
Done.